### ISSUE TITLE
Helm init output does not include Service

### ISSUE THREAD

**Issue by prydonius** [helm/helm#3145]
`helm init -o json/yaml` only outputs the Deployment that will be created. Helm also installs a default Service for Tiller, which can be seen from `helm init --debug --dry-run`. This should be seen in `helm init -o` too.

**Comment by jascott1** [comment]
Actually it was "as designed" because you typically need to do all the jq madness on the deployment and not the service but I did fret about this a bit. Shouldn't be too much work.

**Comment by prydonius** [comment]
Makes sense, the other problem is that I'm not sure how to represent multiple resource types in JSON (in the same way you can do in YAML with the `---` separator).

Another option is to change the help text to explain that this is only for the deployment, as it currently just says "manifest". However, since most people will use this as a preview to see what Helm will install, it would be best to include the Service.
```
-o, --output OutputFormat      skip installation and output Tiller's manifest in specified format (json or yaml)
```

**Comment by ryane** [comment]
I took a shot at addressing this in #3556.

**Comment by ryane** [comment]
there is a pending fix for this in #3556 that's just waiting for a review
